js/category-info.js

```diff
@@ -96,13 +96,15 @@ angular.module('app', [])
                                         //check if repo has unique logo, if not use org logo
                                         var match = false; var name; var file;
                                         for (var f in logos){
-                                            if (logos[f].name == category[count]["name"]){
-                                                match = true; name = logos[f].name ; file = logos[f].file;
+                                            name =  Object.keys(logos[f]);
+                                            if (name == category[count].nameWithOwner.toLowerCase()){
+                                                match = true; file = logos[f][name];
                                             }
                                         }
                                         //if repo has unique logo use it
                                         if (match) {
-                                            category[count]['ownerAvatar'] = "./assets/images/logos/"+name +"-repo-logo." + file;
+                                            category[count]['ownerAvatar'] = "./assets/images/logos/"+category[count]["name"] +"-repo-logo." + file;
```

Comment:
I would actually make the whole filename be the thing that is in the data json, rather than building the name out of the thing and the extension.

So for instance, I'd make the data file:
```json
{
  "data": [
     {"llnl/blt": "blt-repo-logo.png"},
    ...
  ] 
```

And simplify the logic here.

I would also simplify the names... Let's just make a sub-dir in `.../logos` for each org, and then have the filenames be `repo.ext`. That would further change the above to:

```json
{
  "data": [
     {"llnl/blt": "llnl/blt.png"},
    ...
  ] 
```

As an improvement we might also want to just pick a single extension, and get consistent... @hauten , any opinions on what format we should use here? `.png` ? `.jpg` ?

If we do that, then really all we need is a list of the repos that have a custom file, and then loading them is very simple.

Reply:
I rec'd the logo files in various formats, so just passed them along like that. Most are .png, I believe. But I can convert all to one type if needed. Whatever would look & work the best is fine with me.

Reply:
I just don't know which one is best on the web... @LRWeber  / @tzanio have any opinions?

Reply:
Given the use of transparency in several of these logos, my vote would be `.png`.
The logos directly from GitHub's urls are `.png` as well, so it would be consistent.

Reply:
@angfl97 I replaced 6 logo files (they were .jpg or .gif) in that OneDrive folder with transparent .png. They are: Cram, Flux, ROSE, Spack, Umpie, Xbraid. Let me know if you need other options. There are 18 logos files total.

Reply:
So I'll have a sub-directory per repo with that repo's logo inside and in the repo_logos.json file it'll just be a listing of those repo names? and since we know they'll be png I just build out the file name with a .png extension?

Reply:
I think you can do it as sub-dirs for each org, with the files within being the repo name in png format. So `.../logos/<org_name>/<repo_name>.png`

Reply:
Won't the org name always be LLNL? Since these are logos for LLNL owned repos that have a unique logo of their own.

Reply:
I'm imagining there might soon / eventually be repos that are in other orgs that would have dedicated logos. (e.g. projects in the hpc org that we share with LANL, but that we LLNL own).

Reply:
Of the 18 logos I gathered, many are indeed from non-LLNL orgs: CDAT, CEED, ESGF, Flux, MFEM, ROSE, Spack, XBraid, ZFS on Linux. I don't know if all of those logos are uploaded as GH avatars for those orgs or their repos. Let me know if you need me to rename the files differently, get rid of non-LLNL logos, etc. Sorry for any confusion.

Reply:
maybe we have a generic `.../<org_name>/org.png` for the org's avatar? (Though I think really we just pull this from the GitHub API).